eogrow/pipelines/export_maps.py

```diff
@@ -195,9 +199,18 @@ def get_execution_arguments(self, workflow: EOWorkflow) -> List[Dict[EONode, Dic
 
         return exec_args
 
-    def get_geotiff_name(self, name: str) -> str:
-        """Creates a unique name of a geotiff image"""
-        return f"{name}_{self.__class__.__name__}_{self.pipeline_id}.tiff"
+    def get_geotiff_name(self, name: str, crs: Optional[CRS] = None, time: Optional[dt.datetime] = None) -> str:
+        """Creates a name of a geotiff image"""
+        base = f"{self._get_map_name(with_extension=False)}_{name}"
+        if crs is not None:
+            base += f"_UTM_{crs.epsg}"
+        if time is not None:
+            base += f"_{time.strftime(TIMESTAMP_FORMAT)}"
+        return f"{base}.tiff"
+
+    def _get_map_name(self, with_extension: bool = True) -> str:
+        name = self.config.map_name or f"{self.config.feature[1]}.tiff"
+        return name if with_extension else name.replace(".tiff", "")
```

Comment:
perhaps using `os.path.split()[0]` or some variation of it would be better? Not sure how much variation is there in the tiff name, since sometimes it's `.tif` and sometimes `.tiff`, but not sure if it's hard-coded here or not

Reply:
I think there is validation of the input to always be `.tiff` (I checked cause I had a similar concern as you have)

https://github.com/sentinel-hub/eo-grow/blob/ecbd26ba3114c251343afb11300110d8d48c937f/eogrow/pipelines/export_maps.py#L48

Reply:
yeah, fixing it seemed like a good idea at the time
still does :) but I'll switch to the mimetype constant that we have, just to make sure it always matches (which is what Nejc also suggested for map names and he is right)